support-frontend/assets/components/footerCompliant/footerStyles.js

```diff
@@ -2,24 +2,81 @@
 
 import { css } from '@emotion/core';
 import { textSans } from '@guardian/src-foundations/typography';
-import { neutral } from '@guardian/src-foundations/palette';
+import { brand, neutral, brandText, brandAlt } from '@guardian/src-foundations/palette';
+import { from, until } from '@guardian/src-foundations/mq';
 import { space } from '@guardian/src-foundations';
 
 export const componentFooter = css`
-  padding-top: ${space[2]}px;
+  background-color: ${brand[400]};
   ${textSans.small()};
   font-weight: 400;
   color: ${neutral[100]};
   a {
-    color: ${neutral[100]};
-  text-decoration: underline;
-    :hover, :visited {
+    color: ${brandText.anchorPrimary};
+    :hover {
       text-decoration: underline;
-      color: ${neutral[100]};
+      color: ${brandAlt[400]};
     }
```

Comment:
I would try to avoid nesting styles where possible. It's better to attach the styles directly to the element to which they ought to be applied. It makes it easier to update / delete the styles later, as you have a better understanding on the impact.

Reply:
I'm going to leave it in for now as it's applying styling to various links in the legal text passed in, which otherwise show up as browser defaults. Lucy and I were planning to pair on actually rolling this out onto pages next week so I've left a comment to come back to it, as it would definitely be preferable to reduce the specificity here.

Reply:
Yes, it seems like we should restructure this.